migration-engine/connectors/sql-migration-connector/src/lib.rs

```diff
@@ -15,16 +15,7 @@ pub use error::*;
 pub use sql_migration::*;
 
 use migration_connector::*;
-<<<<<<< HEAD
```

Comment:
...

Reply:
been doing a lot of rebasing of the async branch to avoid big conflicts